lib/modules/showImages.js

```diff
@@ -1761,6 +1561,98 @@ addModule('showImages', (module, moduleID) => {
 		}
 	};
 
+	module.videoAdvanced = async function (options) {
```

Comment:
...we wouldn't need to expose this, it could just be `async function videoAdvanced(...`.